commit 3abba74772aa6ffba841418917225c61b6815844 (refs/changes/33/5133/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-11-23T16:26:21-08:00 (11 months ago)
    
    MANTA-4030 many images using node 0.10 or 0.12 broken by transitive lru-cache@4.1.4 dep
    Reviewed by: Angela Fong <angela.fong@joyent.com>
    Approved by: Angela Fong <angela.fong@joyent.com>

diff --git a/package.json b/package.json
index ce30813..588cfb6 100644
--- a/package.json
+++ b/package.json
@@ -1,9 +1,10 @@
 {
     "name": "portolan",
     "description": "SDC Portolan service",
-    "version": "0.1.3",
+    "version": "0.1.4",
     "author": "Joyent (joyent.com)",
     "private": true,
+    "// lru-cache": "attempting to pin version while satisfying transitive deps, because lru-cache@4.1.4 includes yallist@3 which breaks node --abort-on-uncaught-exception with node <v4",
     "dependencies": {
         "assert-plus": "0.1.5",
         "bunyan": "1.8.9",
@@ -13,7 +14,7 @@
         "dtrace-provider": "0.3.1",
         "ipaddr.js": "0.1.6",
         "lomstream": "^1.1.0",
-        "lru-cache": "2.5.0",
+        "lru-cache": "4.1.3",
         "macaddr": "0.0.1",
         "mkdirp": "0.5.0",
         "moray": "3.4.2",
